be/src/storage/rowset/segment_v2/column_reader.h

```diff
@@ -274,6 +274,7 @@ class ColumnReader {
     StarRocksCallOnce<Status> _bloomfilter_index_once;
 
     std::bitset<16> _flags;
+    uint32_t _mem_usage = 0;
```

Comment:
Does a new variable will obey the alignment @sduzh

Reply:
Fixed